### DISCUSSION TITLE
Parallel simulations and Rhino 8 support

### DISCUSSION THREAD

**Discussion by lin-ycv** [Eddy3D-Dev/Discussions#7] [Q&A] [answered]
I'd appreciate it if the devs could answer the following questions:
1. can multiple eddy3d sim/probe be ran simultaneously? would they conflict with each other?
2. is there a version of eddy3d that works with latest version of rhino? or at least 8.12? version 0.4.8.0 doesn't run indoor sim on R8.13

**Comment by kastnerp** [comment, marked as answer]
> can multiple eddy3d sim/probe be ran simultaneously? would they conflict with each other?
Yes, if the names are different, they will not conflict with each other. 

> is there a version of eddy3d that works with latest version of rhino? or at least 8.12? version 0.4.8.0 doesn't run indoor sim on R8.13

Not currently, since McNeel broke the current version with the Rhino 8.9 release. You can download Rhino 8.8 [here](https://files.mcneel.com/dujour/exe/20240611/rhino_en-us_8.8.24163.12481.exe). We plan to release a fix for this over the winter break 2024.

**Comment by kastnerp** [comment]
This has now been fixed!
